src/disclosures/js/views/financial-view.js

```diff
@@ -241,11 +241,14 @@ var financialView = {
       $( '.content_graduate-program' ).hide();
       financialView.gradPlusVisible( false );
     }
-    this.jobPlacementVisible( values.jobRate !== '' );
-    if ( values.level.indexOf( 'Certificate' ) === 1 ) {
-      this.$degreeType.text( 'certificate' );
-    } else {
+    this.jobPlacementVisible(
+      values.jobRate !== 'None' && values.jobRate !== '' &&
+       typeof values.jobRate !== 'undefined'
```

Comment:
Cool. The back-end is supposed to gather up all the undefined values and send them to the API with a 'None' value. But yeah, if something fails on that, the `typeof` check should happen first. Thanks for the catch!